src/main/resources/templates/db_scripts/populateRetainLeadingZeroesSetting.sql

```diff
@@ -0,0 +1,23 @@
+START TRANSACTION;
+
+UPDATE ${myuniversity}_${mymodule}.hrid_settings
```

Comment:
Will it be possible to have a test for the script? You can use `org.folio.rest.api.ItemCopyNumberMigrationScriptTest` as an example

Reply:
Completed in RetainLeadingZeroesMigrationScriptTest.